drivers/gpu/drm/panel/panel-ili7807-fhdplus.c

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0 or later
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
No?

Reply:
https://spdx.org/licenses/

Reply:
Sorry i changed that without thinking too much. The actual issue is with the MODULE_LICENSE

Reply:
I mean it should be changed because the current one is also wrong, but if you replace the spaces with dashes then it should be ok

Reply:
Btw [here](https://www.kernel.org/doc/html/latest/process/license-rules.html) it says that identifier is okay

Reply:
GPL-2.0+ is in the deprecated list on the spdx page I linked. The kernel.org doc should probably be updated...

Reply:
I didn't see that list sorry, gonna change it then. Ty

Reply:
@z3ntu they don't want to mess with updating..

yeah, the only | or-later is modern definition and it's more clear.